client/tos/request.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+export const makeTosAcceptanceRequest = async ( { accept } ) =>
+	apiFetch( {
+		path: '/wc/v3/payments/tos',
+		method: 'POST',
+		data: { accept },
+	} );
+
+export const enableGatewayAfterTosDecline = async () =>
+	apiFetch( {
+		path: '/wc/v3/payments/tos/reactivate',
```

Comment:
It feels a bit strange creating such an endpoint under `/tos` for that. How about reusing `/wc/v3/payments/tos` with `{ accept: false, undo: true }`?

Reply:
The endpoint is for a particular context with some specific logs in place.

I could add this functionality to the upper-level `tos` endpoint, but I don't think that would be better, since the endpoint would be performing two tasks at once:
- Decline the ToS, which might have already been accepted in another tab, for example
- Re-activate the plugin

I think both options are a compromise, but I prefer this one. Do you agree?

Reply:
Ok, let's got with this one.